packages/tinacms/src/toolkit/fields/components/reference/model/reference-field-props.ts

```diff
@@ -1,3 +1,5 @@
+import { CollectionFilters } from '../utils/fetch-optioins-query-builder'
```

Comment:
**question**: Should the imported file be "fetch-options-query-builder"?

Reply:
Did you mean to suggest splitting the types into a separate file? I intentionally kept them in this utils file since they are closely related, making it easier to reference. The utils file is still relatively small, so keeping them together won't cause any mess.

Reply:
Done - fixed file name